roles/init/tasks/postgres.yml

```diff
@@ -65,6 +65,13 @@
   become_user: root
   become: yes
 
+# There are cases where we must behave differently when pg_wal location
+# is inside pgdata. This fact is going to improve code clarity.
+
+- name: Detect if pg_wal location is inside pgdata
+  set_fact:
+    pg_wal_dir_under_pgdata: "{{ true if postgres_wal_dir|dirname == postgres_data_dir else false }}"
```

Comment:
```suggestion
    pg_wal_dir_outside_pgdata: "{{ false if postgres_wal_dir|dirname == postgres_data_dir else true }}"
```

Reply:
Maybe even just `"{{ postgres_wal_dir|dirname != postgres_data_dir }}"`.

Reply:
I would prefer to call it just `pg_wal_outside_pgdata`, but if you want to put in dir, maybe it should be `postgres_wal_dir_outside_pgdata`, to make the correspondence clear.